pages/workspace/publishing.mdx

```diff
@@ -0,0 +1,186 @@
+---
+layout: 'guide'
+title: 'Publishing'
+# abstract: 'Learn about how to publish your Tracking Plan to analytics platforms and APIs'
+---
+
+import Link from '../../components/Link';
+
+<p>
+  <img
+    src={require('../../images/publishing.svg')}
+    width="200px"
+    alt="Avo Publishing"
+/>
+
+</p>
+
+Publishing is where you can configure Avo to publish your Tracking Plan to your analytics platform, a production time validation tool or your own custom endpoint via webhook.
+
+Available publish integrations are currently:
+
+- <a href="#segment-protocols">Segment Protocols</a>
+- <a href="#mixpanel-lexicon">Mixpanel Lexicon</a>
+- <a href="#amplitude-govern">Amplitude Govern</a>
+- <a href="#webhook">Webhook</a>
+
+## Publishing configuration
+
+### Filtering events for publishing
+
+You can customize which events in Avo are published to which publish integrations by using the events filter. Events can be filtered by Sources, Destinations and Tags. For example the following publish integration will only publish events sent from Web to Amplitude.
+
+<center>
+  <img
+    src={require('../../images/publish_filter.png')}
+    width="100%"
+    alt="Publish integration filter to publish only events sent from Web to Amplitude"
+  />
+</center>
+
+### Auto publishing
+
+When Auto Publishing is enabled all changes made on a branch will be published to the integration when the branch is merged.
+
+<center>
+  <img
+    src={require('../../images/auto_publishing.png')}
+    width="100%"
+    alt="Publish integration filter to publish only events sent from Web to Amplitude"
+  />
+</center>
+
+# Publishing integrations
```

Comment:
I find this title pretty confusing and the re-iteration of the list below it unhelpful.

Reply:
Yes I agree, renamed to just "Integrations" in 759cbe4